docs/1.2-DRAFT/metadata.md

```diff
@@ -60,11 +60,11 @@ RO-Crate makes use of the [Linked Data principles] for its description. In parti
 
 RO-Crate realizes these principles using a particular set of technologies and best practices:
 
-1. The _RO-Crate Metadata File_ and _RO-Crate Website_ can be directly published on the web together with the RO-Crate payload. In addition, a data package (e.g. BagIt Zip archive) that contain the RO-Crate can also be published on the web.
-2. The _RO-Crate Metadata File_ is based on the structured data format [JSON-LD]. 
+1. The _RO-Crate Metadata Document_ can be stored in an _RO-Crate Metadata File_ and an _RO-Crate Website_ can be directly published on the web together with the RO-Crate payload. In addition, a data package (e.g. BagIt Zip archive) that contain the RO-Crate can also be published on the web.
```

Comment:
The new version does not say that the metadata file can be published on the web anymore. To preserve the original meaning (assuming that was the intention), it should say:

The _RO-Crate Metadata Document_ can be stored in an _RO-Crate Metadata File_; the _RO-Crate Metadata File_ and an _RO-Crate Website_ can be directly published on the web together with the RO-Crate payload.